src/Data/ProgrammesDb/Type/UtcDateTimeType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Type;
+
+use DateTimeImmutable;
+use DateTimeInterface;
+use DateTimeZone;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\DateTimeType;
+use InvalidArgumentException;
+
+class UTCDateTimeType extends DateTimeType
+{
+    static private $utc;
+
+    public function convertToDatabaseValue($value, AbstractPlatform $platform): ?string
+    {
+        if (!$value instanceof DateTimeInterface) {
+            throw new InvalidArgumentException('$value must implement DateTimeInterface');
+        }
+
+        $utcValue = $value->setTimezone(new DateTimeZone('UTC'));
```

Comment:
if I pass in a \DateTime (not immutable) object, won't this modify it? Or am I being thick?